### PR TITLE
[quest] ADDED: 'Void Plague' Teldrassil Priest Fake Rune Quest To QuestDB

### LINKED ISSUE #5481: [quest] 'Void Plague' Teldrassil Priest Rune Fake Quest Missing From QuestDB
## Description

The SoD Priest fake rune quest 'Void Plague' Teldrassil is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/priest-class-runes-discovery-location
https://www.wowhead.com/classic/spell=425216/engrave-chest-void-plague
https://www.wowhead.com/classic/object=407734/gnarlpine-cache

## Screenshots

N/a

## Questie & WoW version

v9.2.14
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5481
Linked To #5443 

## Proposed changes

- ADDED: 'Void Plague' Teldrassil Priest Fake Rune Quest is now in the QuestDB, and should be accessible to users.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -2589,6 +2589,7 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.questLevel] = 8,
             [questKeys.requiredRaces] = raceIDs.NONE,
             [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Find the Gnarlpine Cache at the bottom of the Ben'ethil Barrow Den and loot if for the rune."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:2592]:
```suggestion
```

File: Database/Corrections/sodQuestFixes.lua [hunk 2]
```diff
@@ -2605,6 +2605,18 @@ function SeasonOfDiscovery:LoadQuests()
...
         },
+        [90157] = {
+            [questKeys.name] = "Void Plague",
+            [questKeys.startedBy] = {nil,{407734}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.PRIEST,
+            [questKeys.objectivesText] = {"Find the Gnarlpine Cache at the bottom of the Ben'ethil Barrow Den and loot if for the rune."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:2616]:
```suggestion
            [questKeys.objectivesText] = {"Find a Gnarlpine Cache at the bottom of the Ben'ethil Barrow Den and loot if for the rune."},
```

**Review by BreakBB — COMMENTED** [review]
This needs to be hidden for now. There are multiple Gnarlpine Caches in the Den.

File: Database/Corrections/sodQuestFixes.lua (hunk 2 above)
Comment (MyTechnoHunter) [reply Database/Corrections/sodQuestFixes.lua:2616]: ...and loot _it_ not _if_ for the rune

Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: This has been fixed in: https://github.com/Questie/Questie/pull/5482/commits/bf2691a373292602e9a8e370d90aa8013ee8cddb

**Review by BreakBB — APPROVED** [review]

**Pull request merged**